espanol/huida/pagar/pagar.md

```diff
@@ -1 +1,5 @@
 Tras aceptar tus dos monedas de oro, el hombre te guia a través de un túnel hasta las afueras de la ciudad.
+Despues te deja salir y cierra la puerta tras de si.
+Estas solo. Ves un camino que se pierde en la distancia hacia el oeste.
+[Decides tomar el camino.](/camino/camino.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.